# const value
byte DEBUG=1 #debug level
byte INFO=2  #general level
byte WARN=4  #warning level
byte ERROR=8 #error level
byte FATAL=16 #fatal/critical level
######### some commit ################
######### some commit ##################
Header header
string str
int8 c
int16 s
int32 i  ############ some commit
int64 l
time t
duration du # some commit
int32[] vi
string[5] strs
std_msgs/Byte[] bytes
std_msgs/Int32[5] int5      # some commit
std_msgs/MultiArrayLayout layout # some commit
### some commit ############################